src/app/utils/keyboard.ts

```diff
@@ -1,5 +1,6 @@
 import { isKeyHotkey } from 'is-hotkey';
 import { KeyboardEventHandler } from 'react';
+import { isTaggedAsComposing } from '../hooks/useSafariCompositionTaggingForKeyDown';
```

Comment:
importing from hooks to utils is discouraged to prevent any unknown circular dependencies formation. will be a good idea to pass the data to this func as arguments.